### ISSUE TITLE
Multiple "Webpack: Starting ..."

### ISSUE THREAD

**Issue by Reeywhaar** [dominique-mueller/simple-progress-webpack-plugin#52]
There are situations when webpack config is a Frankenstein monster which is made of multiple `const base = require("./base.webpack.js")` and so on. So if base config has simple-progress-webpack-plugin in it, message `Webpack: Starting ...` will be spawned even compilation is't started.

https://github.com/dominique-mueller/simple-progress-webpack-plugin/blob/3a4d4cfb4b8de6052ff7cf95fdde50cbdf5a4c46/logger/minimal-logger.js#L13-L26

I suggest to hook into [compilation](https://webpack.js.org/api/compiler-hooks/#compilation) hook and log message there, like
```
compilation.hooks.compilation.tap("SimpleProgressWebpackPlugin", () => {
    log('Webpack: Starting ...');
});
```

Otherwise, it's a great plugin which has advantage to work over ssh or any indirection (like pm2), while other plugins fail. Thanks!

**Comment by chrisabrams** [comment]
I get the same issue as @Reeywhaar has described.

**Comment by Jameskmonger** [comment]
@dominique-mueller are you accepting PRs?

**Comment by dominique-mueller** [comment]
@Jameskmonger I'm happy to accept PRs at any time :)

**Comment by itsdevdom** [comment]
Closed by #62.

**Comment by itsdevdom** [comment]
Released with **[2.0.0](https://github.com/dominique-mueller/simple-progress-webpack-plugin/releases/tag/2.0.0)**!
